src/org/opendatakit/briefcase/export/ExplodeChoiceLists.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.opendatakit.briefcase.export;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.opendatakit.briefcase.reused.Pair;
+
+class ExplodeChoiceLists {
+  static CsvFieldMapper decorate(CsvFieldMapper mapper) {
+    return (localId, workingDir, model, element, configuration) -> {
+      List<Pair<String, String>> output = new ArrayList<>();
+      output.addAll(mapper.apply(localId, workingDir, model, element, configuration).collect(toList()));
```

Comment:
IDEA suggests this:
`List<Pair<String, String>> output = mapper.apply(localId, workingDir, model, element, configuration).collect(toList());`

Reply:
I decided to ignore IntelliJ on this one. I think that it's more clear for any future reader to have an empty list and then some statements that add elements to it as a result of different processes.

Reply:
OK, I can see that.